app/components/ui/breadcrumbs_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav aria-label="Breadcrumbs" class="w-full mb-12 md:mb-0">
+  <ul itemscope itemtype="http://schema.org/BreadcrumbList" class="flex flex-col items-start gap-3 grow flex-wrap sm:flex-row sm:items-center">
+    <% @breadcrumbs.each_with_index do |breadcrumb, i| %>
+      <li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+        <%= link_to breadcrumb[:text],
+          breadcrumb[:path],
+          class: "text-sm text-slate-400 hover:text-slate-200 transition transition duration-150 ease-in-out #{i == @breadcrumbs.size - 1 ? "text-slate-50" : ''}",
```

Comment:
Personally, I think the color for the parent element might be a little bit too dark

![CleanShot 2023-10-21 at 05 07 58](https://github.com/marcoroth/hotwire.io/assets/6411752/490b5fe8-a216-4c00-970a-bf11546099be)

Reply:
Changed the color to match the navigation links on the right top of the site.